### ISSUE TITLE
Default executable name from dubTestTarget can be confusing

### ISSUE THREAD

**Issue by WebFreak001** [atilaneves/reggae#41]
By default the unittests are written into an `ut` executable which could be confusing to a new user.

Maybe calling it `unittest` or `runtest` would be more obvious to what it is.

Alternatively calling it `APPLICATION_NAME-test` by default would be more dub compatible and easier to migrate to for dub users. The dubDefaultTarget right now emits the same executable name as dub after all

**Comment by atilaneves** [comment]
I don't know - I like how it is now and it'd be some work to change it. Also, does anyone run unittest binaries manually with dub? It's usually just `dub test`.

**Comment by WebFreak001** [comment]
Yeah it's just `dub test`, I don't usually run the generated executable but I was confused when that ut file appeared in my folder

**Comment by atilaneves** [comment]
I added some info in the documentation.
